lib/engine/game/g_18_ms.rb

```diff
@@ -62,12 +62,22 @@ def new_operating_round(round_num = 1)
           @phase.next! if @turn == 1 && round_num == 2
         end
 
+        if round_num == 1
+          @players.each do |p|
+            next unless p.cash.negative?
+
+            debt = -p.cash
+            interest = (debt / 2.0 + 0.99).floor
```

Comment:
i still don't quite understand the + 0.99 can you explain to me what your intention is?

Reply:
See comment above.
But the idea was to make interest for debt of $1 to be positive. That is odd debt are rounded up.